code/modules/bitrunning/objects/loot_crate.dm

```diff
@@ -84,6 +89,63 @@
 
 	return TRUE
 
+/obj/structure/closet/crate/secure/bitrunning/decrypted/security
+	name = "decrypted cache"
+	desc = "Compiled from the virtual domain. The reward of a successful Torment Nexus rehabilitation."
+	icon_state = "weaponcrate"
+	base_icon_state = "weaponcrate"
+	locked = TRUE // this has guns in it, make sure it's locked
+	req_access = list(ACCESS_ARMORY)
+
+/obj/structure/closet/crate/secure/bitrunning/decrypted/security/PopulateContents(reward_points, list/completion_loot, rewards_multiplier)
+	spawn_loot(completion_loot)
+	switch(reward_points)
+		if(BITRUNNER_REWARD_MIN)
+			new /obj/item/gun/energy/disabler(src)
+			new /obj/item/gun/energy/disabler(src)
+			new /obj/item/melee/baton/security/loaded(src)
+			new /obj/item/melee/baton/security/loaded(src)
+			new /obj/item/grenade/flashbang(src)
+			new /obj/item/grenade/flashbang(src)
+		if(BITRUNNER_REWARD_LOW)
+			new /obj/item/gun/energy/laser(src)
+			new /obj/item/gun/energy/laser(src)
+			new /obj/item/melee/baton/security/loaded(src)
+			new /obj/item/melee/baton/security/loaded(src)
+			new /obj/item/grenade/flashbang(src)
+			new /obj/item/grenade/flashbang(src)
+			new /obj/item/gun/energy/disabler/smg(src)
+			new /obj/item/gun/energy/disabler/smg(src)
+		if(BITRUNNER_REWARD_MEDIUM)
+			new /obj/item/gun/energy/lasercannon(src)
+			new /obj/item/gun/energy/lasercannon(src)
+			new /obj/item/melee/baton/security/loaded(src)
+			new /obj/item/melee/baton/security/loaded(src)
+			new /obj/item/grenade/frag(src)
+			new /obj/item/grenade/frag(src)
+			new /obj/item/gun/energy/laser/carbine(src)
+			new /obj/item/gun/energy/laser/carbine(src)
```

Comment:
Let me get this straight:

1. Security now has a way to infinity farm frag grenades and laser guns for themselves, at no cost or interaction with cargo, in their down time

and

2. Initially this started as an alternative to brigging with no incentive, meaning it was ultimately up to the security officer (and the prisoner, to a lesser degree) - but now security is incentivized to never use the normal brig, because they are not receiving passive rewards 


Nooooooo, no. No gun or tool rewards in this. You can make it money that gets deposited in the security budget or resources or something.